<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-04T21:07:03 EET" name="Suite" finished-at="2023-11-04T21:07:16 EET" duration-ms="13338">
    <groups>
    </groups>
    <test started-at="2023-11-04T21:07:03 EET" name="Test" finished-at="2023-11-04T21:07:16 EET" duration-ms="13338">
      <class name="com.omayoproject1.centerpagetest.TextAreaFieldTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.omayoproject1.centerpagetest.TextAreaFieldTest@7862f56]" started-at="2023-11-04T21:07:03 EET" name="setUp" finished-at="2023-11-04T21:07:08 EET" duration-ms="4988" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="textAreaFieldTest()[pri:0, instance:com.omayoproject1.centerpagetest.TextAreaFieldTest@7862f56]" started-at="2023-11-04T21:07:08 EET" name="textAreaFieldTest" finished-at="2023-11-04T21:07:12 EET" duration-ms="4222" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- textAreaFieldTest -->
        <test-method is-config="true" signature="closure()[pri:0, instance:com.omayoproject1.centerpagetest.TextAreaFieldTest@7862f56]" started-at="2023-11-04T21:07:12 EET" name="closure" finished-at="2023-11-04T21:07:12 EET" duration-ms="193" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closure -->
      </class> <!-- com.omayoproject1.centerpagetest.TextAreaFieldTest -->
      <class name="com.omayoproject1.centerpagetest.LoginTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.omayoproject1.centerpagetest.LoginTest@3a12c404]" started-at="2023-11-04T21:07:12 EET" name="setUp" finished-at="2023-11-04T21:07:15 EET" duration-ms="3143" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginTest()[pri:0, instance:com.omayoproject1.centerpagetest.LoginTest@3a12c404]" started-at="2023-11-04T21:07:15 EET" name="loginTest" finished-at="2023-11-04T21:07:16 EET" duration-ms="461" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.omayoproject1.centerpageobject.LoginPage.loginWithValidUsernameAndPass(LoginPage.java:32)
at com.omayoproject1.centerpagetest.LoginTest.loginTest(LoginTest.java:16)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="closure()[pri:0, instance:com.omayoproject1.centerpagetest.LoginTest@3a12c404]" started-at="2023-11-04T21:07:16 EET" name="closure" finished-at="2023-11-04T21:07:16 EET" duration-ms="133" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closure -->
      </class> <!-- com.omayoproject1.centerpagetest.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
